authClient/api.go

```diff
@@ -2,12 +2,12 @@ package authClient
 
 // ValidateTokenRequest represents a valid request to the auth service's /validate endpoint.
 type ValidateTokenRequest struct {
-    Token    string `json:"token"`    //The token to validate
-    Internal bool   `json:"internal"` //Whether this token belongs to an internal client
+	Token    string `json:"token"`    //The token to validate
+	Internal bool   `json:"internal"` //Whether this token belongs to an internal client
 }
 
 // ValidateTokenRequest represents a response to calling auth service's /validate endpoint.
 type ValidateTokenResponse struct {
-    ClientId string `json:"client_id"` //The client associated with this token
-    Valid    bool   `json:"valid"`     //Whether the token was valid
+	ClientId string `json:"client_id"` //The client associated with this token
```

Comment:
you're go linter doesn't complain about the claustrophobic comments `// The client ...` ?

Reply:
I started using goland so I wonder if it is different than the vscode one, can update

Reply:
Interestingly it is indifferent to the two. If I run format it won't go to either it just sticks with what is there